### DISCUSSION TITLE
Pellematic alerts : notifications or alerts in homeassistant

### DISCUSSION THREAD

**Discussion by cnico** [dominikamann/oekofen-pellematic-compact#24] [Ideas]
Hi @dominikamann,
I faced an alert this morning because of the trashes tank was full.
The pellematic heater was blocked and I had to manually clean it.

It made me think it would be a nice feature to integrate to the pellematic integration : display an alert in home assistant or send a notification in order to be informed when it arises. 
I did not check by the time but I think the REST API has this alert and should be interpreted.

What do you think of it ?

**Comment by dominikamann** [comment]
Hi @cnico,
you can create a alert by yourself over HA Automations. With the new update the error messages are also available as sensors (Error 1-5).

You are able, for example, to create a push notification to your smart phone (as I did) if an error occurs.

--> https://www.home-assistant.io/docs/automation/basics/

![image](https://user-images.githubusercontent.com/29973737/229916947-b30f9420-32b8-43ee-a2b2-e76b1d9ea91a.png)

![image](https://user-images.githubusercontent.com/29973737/229916928-98ae2302-cbd5-4af5-aedf-da28ed1e5ab1.png)

**Reply by cnico** [reply]
Thanks, I manage to configure it.

**Reply by dominikamann** [reply]
Perfect. May we can create some Blueprint-Automations and share them here in the community. 

https://www.home-assistant.io/docs/automation/using_blueprints/

**Reply by lucaferrucci** [reply]
hello domink, after the last update some sensors give errors again as non-numeric, those of the pellet warehouse for example

**Comment by lucaferrucci** [comment]
good morning, for filling the ash drawer the okofen app also sends the notification.
I advise against using this function, in the settings you can set the drawer capacity, in the case of the compact it is 6kg, I set 5kg so that it warns me before the drawer is full.
Your warning came from the fact that when the auger is full the motor is under load and warns you.
In my opinion this should be avoided, it is better to manage the notification with capacity, avoid soliciting the mechanics.
For the update procedure I'm looking for the document, I apologize but I don't have time to update my machine and film everything for you.
If I can't find the document, I will, I promise

**Reply by cnico** [reply]
I don't see how you can warn before the alert because I do not see any measure of ashes production... and temporaly is not a correct way of doing it because when it is cold the ashes are faster filled...

**Comment by lucaferrucci** [comment]
hello, in the "specialized technician" manual, the update procedure is present in section 26/04, I can attach the Italian version, for translation you can use google translate.
Dominik has access to the site to procure documentation in your language
[PL_BA_IT_Touch_Fachmann_aktuell.pdf](https://github.com/dominikamann/oekofen-pellematic-compact/files/11156718/PL_BA_IT_Touch_Fachmann_aktuell.pdf)

**Reply by dominikamann** [reply]
@lucaferrucci thanks but I think you wrote this in the wrong thread.

**Comment by lucaferrucci** [comment]
I found a beautiful raspberry project made for oekofen, but it's not for homeassistant, it's a standalone project on raspberry PI3, it's called oekofenspy, have you seen it?
I couldn't figure out if it can be integrated into homeassistant with grafana.
It's really well done and captivating

https://gitlab.com/p3605/oekofen-spy/-/blob/main/README.md

**Reply by dominikamann** [reply]
Hi @lucaferrucci,
I already saw this great project. But for me everything needs to be fully integrated into HA. 

You can setup a Influx-DB and Grafana in HomeAssistant. Then you can also build such dashboards. May we can create a new discussion for it and we can share configurations and dashboard there. 

Do you now this possibility already?

https://thesmarthomejourney.com/2021/05/02/grafana-influxdb-home-assistant/

**Comment by lucaferrucci** [comment]
You must enter the code enter the compact setting in the ash extraction menu and you will see the setting

**Reply by cnico** [reply]
I don't have any code to access compact setting.
Can you share pictures of the menu to access ash extraction information?

**Comment by lucaferrucci** [comment]
the password consists of the time and date of the day, in the case of the attachment it is 1607
![Screenshot 2023-04-07 alle 16 46 44](https://user-images.githubusercontent.com/110548623/230628643-ca9f3613-2b6c-494d-acac-12368b219250.jpg)

**Reply by cnico** [reply]
OK, but how do you access this screen?
I do not find it in the menus.

**Reply by Macintras** [reply]
hey @cnico,
in the menu under general you will see a lock at the top right (in version 4.0) that you have to press. If you access via the web interface, the "lock" menu item is at the bottom in the middle,

**Reply by cnico** [reply]
I found it, thanks.
@dominikamann, is there an entity for ashes production usage in the integration ?

**Reply by dominikamann** [reply]
No it is sadly not available on the Ökofen interface. Or at least I was not able to find the values.
